### ISSUE TITLE
implement non-zero @stem.len

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#382]
Verovio currently does not process `@stem.len` for any value other than "0".  It would be useful to process other non-zero values.

Example data with stem length controls on a note:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt>
                <date>2016-12-10 06:55:13</date>
            </pubStmt>
        </fileDesc>
        <encodingDesc>
            <projectDesc>
                <p>Transcoded from Humdrum with Verovio version 0.9.13-dev-66040a8-dirty</p>
            </projectDesc>
        </encodingDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-000000124522585">
                        <staffGrp xml:id="m-000000204341851">
                            <staffDef xml:id="staffdef-000000033029460" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-000000089761626">
                        <measure xml:id="measure-L1" n="0" right="end">
                            <staff xml:id="staff-000000164232734" n="1">
                                <layer xml:id="layer-L1F1" n="1">
                                    <note xml:id="note-L3F1" dur="4" oct="4" pname="c" accid.ges="n" stem.len="0" />
                                    <note xml:id="note-L4F1" dur="4" oct="4" pname="d" accid.ges="n" stem.len="1" />
                                    <note xml:id="note-L5F1" dur="4" oct="4" pname="e" accid.ges="n" stem.len="3" />
                                    <note xml:id="note-L6F1" dur="4" oct="4" pname="f" accid.ges="n" stem.len="5" />
                                    <note xml:id="note-L7F1" dur="4" oct="4" pname="g" accid.ges="n" stem.len="7" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

This is rendered in verovio as:

<img width="284" alt="screen shot 2016-12-10 at 07 01 36" src="https://cloud.githubusercontent.com/assets/3487289/21074154/835ef18e-bea6-11e6-9ad2-6d6d809cec20.png">

But I would expect:

<img width="372" alt="screen shot 2016-12-10 at 07 06 54" src="https://cloud.githubusercontent.com/assets/3487289/21074176/432fb7c8-bea7-11e6-94f4-4012b84a43cb.png">

This feature would allow preserving beam angles when converting SCORE data into verovio, for example (beams should be attached to the ends of `@stem.len` which they currently are not).

Also this would allow me to make chorale-style music look more correct by shortening stems that point away from the staff (until implemented in verovio as an automatic process):

<img width="855" alt="screen shot 2016-12-10 at 07 21 23" src="https://cloud.githubusercontent.com/assets/3487289/21074250/4db977e0-bea9-11e6-8837-87260ba50881.png">

MEI does not allow negative stem directions.  How should placing the stem on the opposite side of the notehead be encoded? (this could be done as negative stem lengths):

<img width="812" alt="screen shot 2016-12-10 at 07 33 57" src="https://cloud.githubusercontent.com/assets/3487289/21074332/12886bc0-beab-11e6-9d33-34bba88d51ae.png">

**Comment by pe-ro** [comment]
MEI records the position of the stem relative to the note head with `@stem.pos`, which can take one of the following values: left, right, or center.

**Comment by lpugin** [comment]
Implemented with e69f8765627dc29c16b2f703d29f7637f12d0da1
